Makefile

```
@@ -3,13 +3,23 @@ SHELL := /bin/bash
+# Workaround for Gentoo Linux "hole in findfunctab" error with race detector
+# See: https://bugs.gentoo.org/961618
+# Gentoo's Go build has issues with the race detector and internal linker.
+# Using external linker resolves the issue.
+ifeq ($(shell test -f /etc/gentoo-release && echo yes),yes)
+    RACE_LDFLAGS := -ldflags="-linkmode=external"
+else
+    RACE_LDFLAGS :=
+endif
 
-.PHONY: build test test-unit test-e2e update watch bench bench-lookup profile flamegraph coverage show-coverage clean lint format prepare-npm generate install-bindings windows windows-x64 windows-arm64 build-windows-cc-image rebuild-windows-cc-image install-git-hooks update-html-attributes vscode-build vscode-package release patch minor major
+.PHONY: build test test-unit test-e2e test-frontend test-frontend-watch test-frontend-update install-frontend update watch bench bench-lookup profile flamegraph coverage show-coverage clean lint format prepare-npm generate install-bindings windows windows-x64 windows-arm64 build-windows-cc-image rebuild-windows-cc-image install-git-hooks update-html-attributes vscode-build vscode-package release patch minor major
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)